sql/hive/src/test/scala/org/apache/spark/sql/sources/SimpleTextHadoopFsRelationSuite.scala

```diff
@@ -69,18 +69,19 @@ class SimpleTextHadoopFsRelationSuite extends HadoopFsRelationTest with Predicat
   test("test hadoop conf option propagation") {
     withTempPath { file =>
       // Test write side
+      val nullVal: String = null
```

Comment:
I am still not pretty sure of this. cc @rxin, could you please check this out if you have some time?